gtest/src/manager/block_exec.rs

```diff
@@ -441,11 +541,36 @@ impl ExtManager {
             gas_multiplier: gsys::GasMultiplier::from_value_per_gas(VALUE_PER_GAS),
             costs: schedule.process_costs(),
             existential_deposit: EXISTENTIAL_DEPOSIT,
-            mailbox_threshold: MAILBOX_THRESHOLD,
+            mailbox_threshold: schedule.rent_weights.mailbox_threshold.ref_time,
             max_reservations: MAX_RESERVATIONS,
             max_pages: TESTS_MAX_PAGES_NUMBER.into(),
             outgoing_limit: OUTGOING_LIMIT,
             outgoing_bytes_limit: OUTGOING_BYTES_LIMIT,
         }
     }
 }
+
+fn get_maximum_task_gas(task: &ScheduledTask<ProgramId>) -> Gas {
+    use ScheduledTask::*;
+    let weights = TaskWeights::default();
+    match task {
+        PauseProgram(_) => Gas(0),
+        #[allow(deprecated)]
+        RemoveResumeSession(_) => Gas(0),
+
+        RemoveFromMailbox(_, _) => Gas(weights.remove_from_mailbox.ref_time),
```

Comment:
These weights must be used in `manager::task` too.
Now `GearCommonGas:::MIN` is returned